faq/sql-faq.md

```diff
@@ -11,6 +11,64 @@ This document summarizes the FAQs related to SQL operations in TiDB.
 
 See [System Variables](/system-variables.md).
 
+## The order of records for a `SELECT` statement without the `ORDER BY` clause is not as same as that in MySQL
+
+It's not a bug. The default order of records depends on various situation without any guarantee of consistency. 
+
+The relevant text in the [SQL-92](http://www.contrib.andrew.cmu.edu/~shadow/sql/sql1992.txt) spec is
+
+> If an `<order by clause>` is not specified, then the table specified by the `<cursor specification>` is T and the ordering of rows in T is implementation-dependent.
+    
+The order of results in MySQL may appear stable because queries are executed in a single thread. However, it's common that query plans can change when upgrading to new versions. It is recommended to use `ORDER BY` whenever an order of results is desired. If without an `ORDER BY`, the two examples below are both acceptable and correct for SQL standard.
```

Comment:
```suggestion
The order of results in MySQL might appear stable because queries are executed in a single thread. However, it is common that query plans can change when upgrading to new versions. It is recommended to use `ORDER BY` whenever an order of results is desired. If without an `ORDER BY`, the following two examples are both acceptable and correct for the SQL standard.
```